_includes/header.html

```diff
@@ -39,7 +39,7 @@
               <li><a href="#welcome">Welcome</a></li>
               <li><a href="#mentors">Mentors</a></li>
               <li><a href="#students">Students</a></li>
-              <li><a href="#projects">Projects</a></li>
+              <li><a href="/projects.html">Projects</a></li>
```

Comment:
this is not related to issue

Reply:
@jogendra Its actually of the master brach of my fork so, it was included. How Can I change that?